### PR TITLE
implement icon parameter for alert rpc

### PR DESCRIPTION
**Pull request by iCollin**
implements [#0177](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0177-alert-icon.md)

### CODE REVIEW

File: src/js/Alert.js [hunk 1]
```diff
@@ -3,16 +3,14 @@ import React from 'react';
 import AlertHeader from './containers/Header';
 import {AlertStrings} from './containers/Metadata'
 import {AlertButtons} from './containers/Buttons';
+import Image from './Templates/Shared/Image'
 
-import alertIcon from '../img/icons/icon-alert.svg'
-
-export default class MediaPlayer extends React.Component {
```
Comment (jacobkeeler) [inline src/js/Alert.js]: Did this work at all before?

**Comment by jacobkeeler** [comment]
Also, should add this image field to UI capabilities

File: src/js/Alert.js (hunk 1 above)
Comment (iCollin) [reply src/js/Alert.js]: yes it did!

File: src/js/Controllers/DisplayCapabilities.js [hunk 2]
```diff
@@ -137,6 +137,13 @@ let capabilities = {
 						"resolutionWidth": 360,
 						"resolutionHeight": 360
 					}
+				},
+				{
+					"name": "alertIcon",
```
Comment (jacobkeeler) [inline src/js/Controllers/DisplayCapabilities.js]: Oddly, `imageTypeSupported` is mandatory in the Mobile API, but not the HMI API. Please include this parameter

Comment (iCollin) [reply src/js/Controllers/DisplayCapabilities.js]: included in https://github.com/smartdevicelink/generic_hmi/pull/177/commits/9e4b4aea1dc8955cee50bf75c6e6b3d2d14ded7d

**Pull request merged**